com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipelineGlobalSettings.cs

```diff
@@ -699,6 +699,12 @@ internal bool AddDiffusionProfile(DiffusionProfileSettings profile)
         [SerializeField]
         internal bool supportProbeVolumes = false;
 
+        /// <summary>
+        /// Controls whether debug display shaders for Rendering Debugger are available in Player builds.
+        /// </summary>
+        [SerializeField]
+        public bool supportRuntimeDebugDisplay = false;
```

Comment:
actually - why is this public and not internal?

Reply:
It's because the original field was public, and I incorrectly thought that the global settings class itself was public (instead of internal). Still, if the class is made public at some point, I see no reason why this shouldn't be public.